Simulation/TSoG_X1_Sim.m

```diff
@@ -27,6 +26,8 @@
                               2 1;
                               15 1];
     TestCase.StopTime = 15;
+
+    TestCase.Plot = @StandardPlots;
```

Comment:
The issue was to remove the plotting from the simulation file entirely. Reason being is if I wanted to create an analysis where I run a large number of tests (or even a couple) then it will be halted or be plotting over eachother every time which is not really useful. 

This line will need to be removed from the simulation file.